rust/opendp/src/trans/mean.rs

```diff
@@ -53,7 +54,7 @@ impl<MI, MO, T> MakeTransformation3<SizedDomain<VectorDomain<IntervalDomain<T>>>
                 IntervalDomain::new(Bound::Included(lower.clone()), Bound::Included(upper.clone()))),
                              length),
             AllDomain::new(),
-            Function::new(move |arg: &Vec<T>| arg.iter().cloned().sum::<T>() / _length),
```

Comment:
It's great that we can get rid of these clones, thanks for figuring out how to do it.